moneyed/test_moneyed_classes.py

```diff
@@ -178,6 +188,8 @@ def test_add(self):
     def test_add_non_money(self):
         with pytest.raises(TypeError):
             Money(1000) + 123
+        custom_class_add = CustomClassesAdd()
```

Comment:
Could you, please move this and the following line to a separate test case?

Reply:
Sure, it's done :+1: